libs/services/designer-client-services/src/lib/standard/search.ts

```diff
@@ -115,7 +116,9 @@ export class StandardSearchService extends BaseSearchService {
       'api-version': apiVersion,
     };
     const response = await httpClient.get<{ value: Connector[] }>({ uri, queryParameters });
-    return [...response.value, ...getClientBuiltInConnectors(filterConnector)];
+    const filteredApiConnectors = filterAPIConnectors(response.value, this.options.apiHubServiceDetails.subscriptionId !== undefined);
```

Comment:
We should pull out the `isAzureConnectorsEnabled` as
```suggestion
const isAzureConnectorsEnabled = this.options.apiHubServiceDetails.subscriptionId !== undefined;
if (isAzureConnectorsEnabled) response.value = filterAPIConnectors(response.value)
```
to avoid any unnecessary filtering operations (will be faster, and won't run the filtering in portal)